app/serializers/search_document_api_serializer.rb

```diff
@@ -0,0 +1,33 @@
+class SearchDocumentApiSerializer < ActiveModel::Serializer
+  attributes :searchable_id, :searchable_type, :searchable, :url, :project, :project_url, :story, :story_url, :attachment_image
+
+  def project_url
+    project.present? ? api_v1_project_path(project) : nil
```

Comment:
Does our search document preload project and story so this query isn’t going to N+1 ?